biopet-core/src/main/resources/nl/lumc/sasc/biopet/utils/rscript/plotScatter.R

```diff
@@ -1,40 +0,0 @@
-library(reshape2)
```

Comment:
yes, those 3 script where duplicated in biopet-core while they already exist in biopet-utils.